modules/machineset-yaml-rhv.adoc

```diff
@@ -93,39 +95,45 @@ The `instance_type_id` field is deprecated and will be removed in a future relea
 ====
 +
 If you include this parameter, you do not need to specify the hardware parameters of the VM including CPU and memory because this parameter overrides all hardware parameters.
+<9> Setting this option to `false` enables preallocation of disks. The default is `true`. Setting `sparse` to `true` with `format` set to `raw` is not available for block storage domains. The 'raw' format writes the entire virtual disk to the underlying physical disk.
+<10> Can be set to `cow` or `raw`. The default is `cow`. The 'cow' format is optimized for virtual machines.
```

Comment:
The same applies for one of the instances of the "cow" code reference. Please change it from single quotes to back-ticks.